RRemoteConfig/Fetcher.swift

```diff
@@ -13,14 +13,14 @@ internal class Fetcher {
             return completionHandler(nil)
         }
         var request = URLRequest(url: url)
-        request.addHeader("apiKey", "ras-\(environment.subscriptionKey)")
-        request.setExtraHeaders(from: environment)
+        request.setConfigHeaders(from: environment)
 
         apiClient.send(request: request, decodeAs: ConfigModel.self) { (result, response) in
             switch result {
             case .success(let resultConfig):
                 var config = resultConfig as? ConfigModel
                 config?.signature = response?.allHeaderFields["Signature"] as? String
+                self.environment.etag = response?.allHeaderFields["Etag"] as? String
```

Comment:
What happens here on a `304` response? The `304` response doesn't have the ETag header, so will this still keep the ETag cached?

Reply:
I actually get a http status `200` response from the URLSession's url cache.
For the same request/response Charles shows the expected `304`.
Presumably URLSession's cache sends back the entire previous response.

Reply:
Ah, okay that makes sense.